app/code/Magento/CatalogImportExport/Model/StockItemImporterInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CatalogImportExport\Model;
+
+/**
+ * Interface StockItemImporterInterface
+ *
+ * @api
+ */
+interface StockItemImporterInterface
```

Comment:
there is no default implementation of this interface in CE, just in MSI

that will lead to Fatal Error

Reply:
https://github.com/magento-engcom/msi/issues/496

Reply:
Done